src/Maatwebsite/Excel/Classes/LaravelExcelWorksheet.php

```diff
@@ -39,25 +39,25 @@ class LaravelExcelWorksheet extends PHPExcel_Worksheet {
      * View
      * @var string
      */
-    public $view;
+    private $view;
```

Comment:
The revert of private -> public was meant for all changes, not just the first one.